### ISSUE TITLE
Several problems with Meteorite

### ISSUE THREAD

**Issue by Bankaifan** [cellcortex/homebridge-yeelighter#71]
Hi, I got some problems with the Meteorite.

First thing is - how can I rename the Background-Light?
![image](https://user-images.githubusercontent.com/476059/106350395-6fe88e80-62d5-11eb-8d98-57cd8b1ba822.png)
In my Home-App it is called "0x00000000124d9007 background" what makes it impossible to control it via Siri.

I tried to override its name, but does not work:
```
[
    {
        "id": "0x00000000124d9007#background",
        "name": "Meteorite Background",
        "offOnDisconnect": true,
        "nightLight": true
    }
]
```

The other thing is, when I pull down the brightness till zero, I get an error

> 30.1.2021, 08:26:07 [Yeelighter] [0x00000000124d9007#background] set brightness [ 42 ]
> 30.1.2021, 08:26:07 [Yeelighter] [0x00000000124d9007#background] set brightness [ 38 ]
> 30.1.2021, 08:26:07 [Yeelighter] [0x00000000124d9007#background] set brightness [ 100 ]
> 30.1.2021, 08:26:08 [Yeelighter] [0x00000000124d9007#background] set brightness [ 31 ]
> 30.1.2021, 08:26:08 [Yeelighter] [0x00000000124d9007#background] set brightness [ 0 ]
> 30.1.2021, 08:26:08 [Yeelighter] [0x00000000124d9007] Error returned for request [24]: {"code":-5001,"message":"invalid params"} []
> 30.1.2021, 08:26:08 (node:18816) UnhandledPromiseRejectionWarning: #<Object>
> 30.1.2021, 08:26:08 (node:18816) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
> 30.1.2021, 08:26:08 (node:18816) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
> 30.1.2021, 08:26:14 [Yeelighter] [0x00000000124d9007] temperature light updated

**Comment by Bankaifan** [comment]
Also, if I update the Color of the Backgroundlight via Siri, it does not get updated in Home-App

**Comment by mariorasch** [comment]
HI there!

I am also using a Meteorite (ceiling10) lamp. After adding Yeelighter plugin and restarting Homebridge service I can see the log entries below. In HomeKit two devices are added named "Meteorite" and "0x0000000001234567 background" and everything works fine after that.

But when I rename these devices in HomeKit and restart Homebridge service it is not possible any more to control the devices. I could only fix this by removing the device from Homebridge's cache, restarting Homebridge service and then switching off and on "LAN Control" for my lamp.

Is there any chance you can fix this so that it is possible to rename the devices in HomeKit?

Many thanks in advance & best regards!
Mario.

```
[23/05/2021, 13:22:54] [Yeelighter] New ceiling10 [0x0000000001234567] found at yeelight://1.2.3.4:12345
[23/05/2021, 13:22:54] [Yeelighter] Loading accessory from cache: 0x0000000001234567 (ceiling10)
[23/05/2021, 13:22:54] [Yeelighter] Accessory created with UUID cd85ba26-6ea3-4bdc-9f4e-ca1db9089a7a
[23/05/2021, 13:22:54] [Yeelighter] [0x0000000001234567] no subtype []
[0x0000000001234567@@Configured Name] Characteristic not in required or optional characteristic section for service Lightbulb. Adding anyway.
[23/05/2021, 13:22:54] [Yeelighter] [0x0000000001234567#background] registering subtype 0x0000000001234567#background []
[0x0000000001234567@0x0000000001234567 background@Configured Name] Characteristic not in required or optional characteristic section for service Lightbulb. Adding anyway.
[23/05/2021, 13:22:54] [Yeelighter] [0x0000000001234567] installed as Color temperature light with ambience light []
[23/05/2021, 13:22:54] [Yeelighter] [0x0000000001234567] ceiling10 Connected []
[23/05/2021, 13:22:54] [Yeelighter] [0x0000000001234567] temperature light updated {"power":true,"color_mode":2,"bright":100,"hue":0,"sat":0,"ct":4000,"bg_power":true,"bg_bright":100,"bg_hue":343,"bg_sat":100,"bg_ct":5376,"bg_lmode":3,"nl_br":0,"active_mode":0,"name":""} []
[23/05/2021, 13:22:56] [Yeelighter] New (cached) ceiling10 [0x0000000001234567] found at yeelight://1.2.3.4:12345
[23/05/2021, 13:23:00] [Yeelighter] New (cached) ceiling10 [0x0000000001234567] found at yeelight://1.2.3.4:12345
```

**Comment by djchoyr** [comment]
I have always had similar problems. I have Homebridge installed on a raspberry. Normally it worked well. But sometimes until I reinstall all Homebridge from scratch I have no way to make it work again. In the Homebridge registry I can see that it locates a lamp in the cache. When I try to turn on the lamp from Homebridge or the home app, it doesn't do anything. But if I use the Yeelight app it does work perfectly.

**Comment by ocywob** [comment]
I'm also using Crystal Pendandt Light from Yeelight and when I try to reduce brightness to 0, homebridge crashes.

![image](https://user-images.githubusercontent.com/80576812/149806237-2443fa07-ce7f-41af-a841-24fc852b5155.png)

`[17/01/2022, 16:21:17] [Masa Lamba] [0x00000000157f3c0a] set brightness [ 0 ]
[17/01/2022, 16:21:17] [Masa Lamba] [0x00000000157f3c0a] Error returned for request [30]: {"code":-5001,"message":"invalid params"} []
[17/01/2022, 16:21:17] UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<Object>".`

**Comment by cellcortex** [comment]
this should be fixed with 2.4.x
